app/controllers/component_templates_controller.rb

```diff
@@ -13,19 +13,22 @@ def new
 
   def edit
     authorize @component_template
-    @component_attributes = JSON.pretty_generate(@component_template.component_attributes)
+    @source = JSON.pretty_generate(@component_template.source)
+    @bootstrappers = JSON.pretty_generate(@component_template.bootstrappers)
   end
 
   def show
     authorize @component_template
-    @component_attributes = JSON.pretty_generate(@component_template.component_attributes)
+    @bootstrappers = JSON.pretty_generate(@component_template.bootstrappers)
+    @source = JSON.pretty_generate(@component_template.source)
   end
 
   def create
     authorize ComponentTemplate
-    component_attributes = component_template_params.clone
-    component_attributes[:component_attributes] = JSON.parse(component_attributes[:component_attributes])
-    @component_template = ComponentTemplate.new(component_attributes)
+    ct_params_cloned = component_template_params.clone
```

Comment:
Noted for the avoidance of using abbreviation. 
We cloned the object from inputs because we need to JSON parse it first before adding it to database.